### PR TITLE
The one about the Jakarta JAX-B ContextFactory 

### PR DESCRIPTION
**Pull request by neuwerk**

### CODE REVIEW

**Comment by neuwerk** [comment]
#build

File: dev/com.ibm.websphere.appserver.features/visibility/public/jaxb-3.0/io.openliberty.jaxb-3.0.feature [hunk 1]
```diff
@@ -21,6 +21,6 @@ Subsystem-Name: Jakarta XML Bindings 3.0
   com.ibm.websphere.appserver.eeCompatible-9.0, \
   io.openliberty.jakarta.jaxb-3.0
 -bundles=\
-  io.openliberty.jaxb.3.0.internal.tools
+  io.openliberty.jaxb.internal.tools.3.0
```
Comment (tevans78) [inline dev/com.ibm.websphere.appserver.features/visibility/public/jaxb-3.0/io.openliberty.jaxb-3.0.feature:24]: I think this was named correctly, why change it?

File: dev/io.openliberty.jaxb.internal.tools.3.0/.project [hunk 2]
```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <projectDescription>
-	<name>io.openliberty.jaxb.3.0.internal.tools</name>
+	<name>io.openliberty.jaxb.inertnal.tools.3.0</name>
```
Comment (tevans78) [inline dev/io.openliberty.jaxb.internal.tools.3.0/.project:3]: Even if changing the name really is required, you have a typo here

File: dev/io.openliberty.jakarta.jaxb.3.0/src/jakarta/xml/bind/ModuleUtil.java [hunk 3]
```diff
@@ -0,0 +1,65 @@
+/*
```
Comment (tevans78) [inline dev/io.openliberty.jakarta.jaxb.3.0/src/jakarta/xml/bind/ModuleUtil.java:1]: I assume this is a temporary overlay?

File: dev/io.openliberty.jaxb.internal.tools.3.0/bnd.bnd [hunk 4]
```diff
@@ -11,10 +11,10 @@
 -include= ~../cnf/resources/bnd/bundle.props
 
 bVersion=1.0.${libertyBundleMicroVersion}
-bSymbolicName=io.openliberty.jaxb.3.0.internal.tools
+bSymbolicName=io.openliberty.jaxb.internal.tools.3.0
```
Comment (tevans78) [inline dev/io.openliberty.jaxb.internal.tools.3.0/bnd.bnd:14]: See here: https://github.com/OpenLiberty/open-liberty/wiki/Open-Liberty-naming-guidelines#examples
- Internal project/bundle name = io.openliberty.example.1.0.internal.subname

File: dev/io.openliberty.jakarta.jaxb.3.0/src/jakarta/xml/bind/ModuleUtil.java (hunk 3 above)
Comment (neuwerk) [reply dev/io.openliberty.jakarta.jaxb.3.0/src/jakarta/xml/bind/ModuleUtil.java:1]: @tevans78 that's correct, without an update to the ContextFactory to point to the Glassfish RI, the JPA feature will hit a NCDFE as the Jakarta JAX-B spec will keep trying to find the original Sun RI. I'll passing this change on to the Jakarta JAX-B project.

File: dev/com.ibm.websphere.appserver.features/visibility/public/jaxb-3.0/io.openliberty.jaxb-3.0.feature (hunk 1 above)
Comment (neuwerk) [reply dev/com.ibm.websphere.appserver.features/visibility/public/jaxb-3.0/io.openliberty.jaxb-3.0.feature:24]: @tevans78 so I may be misremembering, but I thought that the ResourceProvider had to match the bundle name exactly (with the exception of using under scores for the versioning). As a part of getting the feature working correctly, I changed the bundle name to match the ResourceProvider.

This is what it is presently:
```
Service-Component: \
  io.openliberty.jaxb.internal.tools_3_0_${replace;${bVersion};\\.;_}.ResourceProvider; \
```
I attempted this locally, and the Jakarta ContextFactory Service still wasn't getting picked up:
```
Service-Component: \
  io.openliberty.jaxb_3_0_internal.tools_${replace;${bVersion};\\.;_}.ResourceProvider; \
```

**Comment by neuwerk** [comment]
#build

**Comment by neuwerk** [comment]
#build

**Comment by andrea-pichler** [comment]
No L2 message review is needed for translations

**Comment by neuwerk** [comment]
#run-libby-bot

**Comment by neuwerk** [comment]
#build

**Comment by neuwerk** [comment]
#run-libby-bot

**Pull request merged**